src/parquet/murmur3.cc

```diff
@@ -411,42 +192,34 @@ void Hash_x64_128(const void* key, const int len, const uint32_t seed,
 template <typename T>
 uint64_t HashHelper(T value, uint32_t seed) {
   HashOutput128 output;
-  Hash_x64_128(reinterpret_cast<void*>(&value), sizeof(T), seed, &output);
+  Hash_x64_128(reinterpret_cast<void*>(&value), sizeof(T), seed, output.val);
```

Comment:
I'd suggest removing the special type and declaring `uint64_t output[2];` here and below.